### DISCUSSION TITLE
Variable questions

### DISCUSSION THREAD

**Discussion by raystek** [kimai/kimai#2280] [Questions & Answers]
I’m newbie to kimai2 and have a questions about the variables used in twig templates. Where are variables such as model.template.title defined? I am wanting to change values. Thanks for any advice.

**Comment by kevinpapst** [comment]
In the invoice template definition in Kimai at /en/invoice/template

**Reply by raystek** [reply]
Sorry but I can't find the path /en/invoice/template in my kimai2 directory

**Reply by kevinpapst** [reply]
Its a URL - you define the title of your invoice via the GUI

**Reply by raystek** [reply]
Found it. Thank you. Is it possible to customize the variables? For example, the current value of label.date is "Date".  I'd like to change it to "Invoice Date".

**Reply by kevinpapst** [reply]
See https://www.kimai.org/documentation/branding.html - Translations can be overwritten by your own files.
Or you simply copy & paste the used invoice template to var/invoices/ and change its content.
